contract_variable_quantity/views/contract_line_formula.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_contract_line_qty_formula_tree" model="ir.ui.view">
+        <field name="model">contract.line.qty.formula</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+            </tree>
+        </field>
+    </record>
+    <record id="view_contract_line_qty_formula_form" model="ir.ui.view">
+        <field name="model">contract.line.qty.formula</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <div class="oe_title">
+                        <h1>
+                            <field name="name" placeholder="Name" />
+                        </h1>
+                    </div>
+                    <group string="Code">
+                        <div style="margin-top: 4px;">
```

Comment:
This is displaying weird, please review it, something must be changed, review with others module version